packages/svelte/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nxext/svelte",
-  "version": "20.0.3",
+  "version": "0.0.1-e2e",
```

Comment:
That looks like something the e2e tests did. Could you roll back the version changes?